test/run_tests

```diff
@@ -69,6 +69,11 @@ parser.add_argument(
     help="Controls verbose failure"
 )
 
+parser.add_argument(
+   "--mpicmd", default="mpiexec -n ", type=str, required=False,
```

Comment:
Do you want to do `--mpi-cmd` instead? Usually multi-word switches have some sort of separation and most commonly used (from what I have seen) was `-`.

Reply:
I think that's going to fail with argparse (at least with the current implementation).  Let me check to make sure.

Reply:
Aha... the trick is that "-" gets replaced by "_" once argparse parses the input.  Done!

Reply:
will argpase always behave like this int he future? why not change right away to `--mpi_cmd` then ?

Reply:
They also have `dest=` param, so in case something breaks, we just do `dest='mpi_cmd'`.

Reply:
> will argpase always behave like this int he future? why not change right away to `--mpi_cmd` then ?

It will... that just internally though.  For user facing input, I think I do prefer `-- mpi-cmd`.